ARVideoKit/Sources/ARView.swift

```diff
@@ -96,54 +96,54 @@ import ARKit
         parentVC = vc
     }
     
-    @objc private func deviceDidRotate() {
-        guard var views = parentVC?.view.subviews else {
-            return
-        }
-        
-        var rotationAngle = 0
-        
-        switch inputViewOrientationMode {
-        case .auto:
-            views = views.filter { !$0.isARView && $0.isButton }
-        case .all:
-            views = views.filter { !$0.isARView }
-        case .manual(let subviews):
-            views = subviews.filter { !$0.isARView }
-        case .disabled:
-            views = []
-        }
-        var angleEnabled: Bool {
-            for v in inputViewOrientations {
-                if UIDevice.current.orientation.rawValue == v.rawValue {
-                    return true
-                }
-            }
-            return false
-        }
-        switch UIDevice.current.orientation {
-        case .landscapeLeft:
-            rotationAngle = 90
-            recentAngle = 90
-        case .landscapeRight:
-            rotationAngle = -90
-            recentAngle = -90
-        case .faceUp, .faceDown, .portraitUpsideDown:
-            rotationAngle = recentAngle
-        default:
-            rotationAngle = 0
-            recentAngle = 0
-        }
-
-        if !angleEnabled {
-            rotationAngle = 0
-        }
-        
-        for view in views {
-            UIView.animate(withDuration: 0.2, animations: {
-                view.transform = CGAffineTransform(rotationAngle: CGFloat(rotationAngle).degreesToRadians)
-            })
-        }
-        
-    }
+//    @objc private func deviceDidRotate() {
+//        guard var views = parentVC?.view.subviews else {
+//            return
+//        }
+//        
+//        var rotationAngle = 0
+//        
+//        switch inputViewOrientationMode {
+//        case .auto:
+//            views = views.filter { !$0.isARView && $0.isButton }
+//        case .all:
+//            views = views.filter { !$0.isARView }
+//        case .manual(let subviews):
+//            views = subviews.filter { !$0.isARView }
+//        case .disabled:
+//            views = []
+//        }
+//        var angleEnabled: Bool {
+//            for v in inputViewOrientations {
+//                if UIDevice.current.orientation.rawValue == v.rawValue {
+//                    return true
+//                }
+//            }
+//            return false
+//        }
+//        switch UIDevice.current.orientation {
+//        case .landscapeLeft:
+//            rotationAngle = 90
+//            recentAngle = 90
+//        case .landscapeRight:
+//            rotationAngle = -90
+//            recentAngle = -90
+//        case .faceUp, .faceDown, .portraitUpsideDown:
+//            rotationAngle = recentAngle
+//        default:
```

Comment:
@seriyvolk83 I'd suggest, instead of just commenting out the option of handling the orientation to have a boolean/method that would disable that functionality. There may be some developers out there who would like the option to disable/enable it instead of having it eliminated completely

Reply:
Ok. I'll add it.

Reply:
@AFathi  Can this changes be merge. I am really facing this issue with orientation.

Reply:
@AFathi  The Video Render is working perfectly but only in portrait.